python/cudf_polars/cudf_polars/experimental/benchmarks/tpch.py

```diff
@@ -963,88 +1156,84 @@ def run(args: Any) -> None:
             except ImportError as err:
                 if args.shuffle == "rapidsmpf":
                     raise ImportError from err
-        broadcast_join_limit = args.broadcast_join_limit or 2
+        broadcast_join_limit = args.broadcast_join_limit
```

Comment:
```suggestion
        broadcast_join_limit = run_config.broadcast_join_limit
```
We run into a problem when `args.broadcast_join_limit` is None. Do we need the other `args.*` above to come from `run_config.` as well?

Reply:
Good call.

I think moving everything over to `run_config` is probably best. I'll do that now.

Reply:
@rjzamora https://github.com/rjzamora/cudf/pull/3 fixes that.